### PR TITLE
add outline-color to button, textarea, input

### PR DESCRIPTION
**Pull request by BurningTreeC**

### CODE REVIEW

**Comment by Jermolene** [comment]
Thanks @BurningTreeC I'm nervous about including a colour that isn't defined in the core palettes because it makes it so hard for people to find. I also wonder whether "outline" is the right name; we use "border" for other things. Perhaps it should be "control-border"?

File: themes/tiddlywiki/vanilla/base.tid [hunk 1]
```diff
@@ -417,6 +421,10 @@ a.tc-tiddlylink-external:hover {
 ** Buttons
 */
 
+button {
```
Comment (Jermolene) [inline themes/tiddlywiki/vanilla/base.tid]: Even though this is about buttons, it should go up at the top where we have gathered the other style declarations that target all instances of HTML elements.

**Comment by BurningTreeC** [comment]
>  I'm nervous about including a colour that isn't defined in the core palettes because it makes it so hard for people to find. I also wonder whether "outline" is the right name; we use "border" for other things. Perhaps it should be "control-border"?

should we just use the primary color?

File: themes/tiddlywiki/vanilla/base.tid (hunk 1 above)
Comment (BurningTreeC) [reply themes/tiddlywiki/vanilla/base.tid]: ok, I'll move it

**Comment by Jermolene** [comment]
The primary colour is supposed to be a prominent highlight colour. Perhaps the "muted-foreground" colour?

**Comment by BurningTreeC** [comment]
> Perhaps the "muted-foreground" colour?

I see the outlining mostly used to highlight the element that has focus, so the primary color would match for that purpose.

For me it should just match the palette and I'm fine with muted-foreground

**Comment by Jermolene** [comment]
> I see the outlining mostly used to highlight the element that has focus, so the primary color would match for that purpose.

I'd misunderstood; the primary colour would indeed be a reasonable choice.

**Comment by BurningTreeC** [comment]
changed to use primary @Jermolene

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**